server/src/main/java/org/apache/iotdb/db/mpp/plan/execution/config/executor/IConfigTaskExecutor.java

```diff
@@ -130,4 +133,12 @@ SettableFuture<ConfigTaskResult> showPathSetTemplate(
 
   SettableFuture<ConfigTaskResult> deleteTimeSeries(
       String queryId, DeleteTimeSeriesStatement deleteTimeSeriesStatement);
+
+  SettableFuture<ConfigTaskResult> getRegion(GetRegionStatement getRegionStatement);
```

Comment:
getRegion or getRouting?

Reply:
After the newest discussion, the new function names regarding getting regions are all renamed, yet the old ones remain the same. Do I have to change them?

Reply:
getRegionId is better?